### PR TITLE
/check endpoint for healthcheck pings

### PR DESCRIPTION
**Pull request by tonyto**
### Context

PaaS requires a healthcheck endpoint

### Changes proposed in this pull request

Adding a `/check` route that will respond with the following as an example

```
{"status":"OK","version":SHA,"environment":"development"}
```

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,6 +1,10 @@
 class ApplicationController < ActionController::Base
   layout :layout_by_resource
 
+  def check
+    render json: { status: "OK", version: ENV["SHA"], environment: Rails.env }, status: :ok
+  end
```
Comment (guy-roberts) [inline app/controllers/application_controller.rb:6]: Does the json include the Rails environment ?  Is there anything in there that would be useful to bad people ?

**Review by guy-roberts — COMMENTED** [review]
Looks good to me

Just a question, which will not stop the approval, about if the Rails env is shared in the JSON response ?

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (marcrice12) [reply app/controllers/application_controller.rb:6]: looks like a simple json response object: status string, encrypted version string (SHA), string saying the environment: "production" or "development"

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,6 +1,10 @@
 class ApplicationController < ActionController::Base
   layout :layout_by_resource
 
+  def check
+    render json: { status: "OK", version: ENV["SHA"], environment: Rails.env }, status: :ok
```
Comment (marcrice12) [inline app/controllers/application_controller.rb:5]: where does ENV["SHA"] originate?

Comment (tonyto) [reply app/controllers/application_controller.rb:5]: It'll be set when the app is deployed. If it's not set, it'll be `nil`

**Comment by tonyto** [comment]
@guy-roberts `Rails.env` will have one of the follow values `development`, `test, `staging`, `production`

**Review by guy-roberts — APPROVED** [review]

**Pull request merged**